tests/features/page-header.feature

```diff
@@ -78,6 +78,16 @@ Feature: Page header block component.
     # The default text format should be applied, converting URLs into links.
     And I should see the link "http://www.example.org" in the "page header intro"
 
+  Scenario: Project content type has custom metadata shown in the page header.
```

Comment:
this could have been done also here https://github.com/openeuropa/oe_theme/blob/EWPP-181/tests/Functional/ContentProjectRenderTest.php#L145, but since we have this feature I guess its ok.
@ademarco what do you think?

Reply:
Yep, we need to use functional tests from now on. Lets move it there.